### ISSUE TITLE
Version 1.21.0 stopped working on Opera

### ISSUE THREAD

**Issue by dellagustin** [podStation/podStation#63]
Even though Opera is not officially supported, I discovered after releasing 1.21.0 that I actually do have Opera users, and that Opera does support the installation for Chrome Extensions 🎉 🎈 🎂 

Now the reason I discovered it, is because a user reported that it stopped working on Opera 😢 
> My podcast feed is suddenly empty and the buttons to subscribe no longer work.  I have tried disabling, then reenabling as well as uninstalling and reinstalling.  Is there anything that can be done to fix this?

I will check what I can do.

**Comment by dellagustin** [comment]
I installed Opera and managed to install the extension.
First thing I noticed is that I can't add podcasts to it from the search results (did not check other means yet).

Inspecting the background page I get the following errors:
```
angular.js:4442 Uncaught TypeError: Cannot read property 'setActionHandler' of undefined
    at Object.audioPlayerService (audioPlayer.js:547)
    at Object.invoke (angular.js:4478)
    at Object.enforcedReturnValue [as $get] (angular.js:4330)
    at Object.invoke (angular.js:4478)
    at angular.js:4295
    at getService (angular.js:4437)
    at Object.invoke (angular.js:4469)
    at angular.js:4299
    at forEach (angular.js:336)
    at createInjector (angular.js:4299)
reuse.js:23 Uncaught Error: could not get service "messageService", inke 
    at getAngularService (reuse.js:23)
    at getMessageService (reuse.js:30)
    at registerMessageListeners (podcastManager.js:448)
    at HTMLDocument.<anonymous> (podcastManager.js:442)
    at j (jquery.min.js:2)
    at Object.fireWith [as resolveWith] (jquery.min.js:2)
    at Function.ready (jquery.min.js:2)
    at HTMLDocument.I (jquery.min.js:2)
3background.html:1 Error in event handler: TypeError: Cannot read property 'paused' of undefined
    at buildAudioInfo (chrome-extension://bpcagekijmfcocgjlnnhpdogbplajjfn/background/entities/audioPlayer.js:205:26)
    at chrome-extension://bpcagekijmfcocgjlnnhpdogbplajjfn/background/entities/audioPlayer.js:509:17
    at chrome-extension://bpcagekijmfcocgjlnnhpdogbplajjfn/reuse/messageServiceDefinition.js:42:19
    at Array.forEach (<anonymous>)
    at dispatchToListeners (chrome-extension://bpcagekijmfcocgjlnnhpdogbplajjfn/reuse/messageServiceDefinition.js:41:33)
    at chrome-extension://bpcagekijmfcocgjlnnhpdogbplajjfn/reuse/messageServiceDefinition.js:89:12
```

I am quite sure the culprit is 4fb90ebf0d72f82724f007f460ffd1592a69c2e0.
Apparently Opera does not support mediaSession yet.
I need to include Opera automated tests on my pipeline.
I have to confess I was a bad developer and did not add automated tests for the mediaSession support, but, hey who does not like to live dangerously 😁 ...

**Comment by dellagustin** [comment]
Solved with d5a9bda30d00921a6461b1204e1f03dfc35110f7.
Version 1.22.1 is released, still pending review from Chrome Webstore, as soon as it is published podStation should work again on Opera.
